tests/block_test.go

```diff
@@ -63,7 +63,6 @@ func TestExecutionSpecBlocktests(t *testing.T) {
 		t.Skipf("directory %s does not exist", executionSpecBlockchainTestDir)
 	}
 	bt := new(testMatcher)
-	bt.skipLoad(".*prague.*") // TODO(Nathan): remove it before enable prague
```

Comment:
There is no `prague` dir in the following test case set
https://github.com/ethereum/execution-spec-tests/releases/download/v2.1.0/fixtures_develop.tar.gz